autoload/miniSnip.vim

```diff
@@ -62,7 +62,13 @@ function! s:directories() abort
 endfunction
 
 function! s:findSnippetFile() abort
-  let cword = matchstr(getline('.'), '\v\f+%' . col('.') . 'c')
+  if exists("g:miniSnip_expandpattern")
+    let expandpattern = s:getVar("expandpattern")
+  else
+    let expandpattern = '\v\f+%'
+  endif
+  let cword = matchstr(getline('.'), expandpattern . col('.') . 'c')
```

Comment:
Very important parts of pattern are missing - those cannot be left for the user to include in their custom patterns (even if we explicitly document such need in the help page).  
It should be `expandpattern . '\v%' . col('.') . 'c'`